File: Sesion01/Ejercicios/E1.cpp
#include <iostream>
using namespace std;


int sumarElementos(int arreglo[], int tamano) {
    int suma = 0;
    for (int i = 0; i < tamano; ++i) {
        suma += arreglo[i];
    }
    return suma;
}


int main() {
    int arreglo[] = {1, 2, 3, 4, 5};
    int tamano = sizeof(arreglo) / sizeof(arreglo[0]);
    int resultado = sumarElementos(arreglo, tamano);
    cout << "La suma de los elementos es: " << resultado << endl;
    return 0;
}

File: Sesion01/Ejercicios/E1.java
package s1;

public class e1 {
	public static int sumarElementos(int[] arreglo) {
       int suma = 0;
       for (int num : arreglo) {
           suma += num;
       }
       return suma;
   }


   public static void main(String[] args) {
       int[] arreglo = {1, 2, 3, 4, 5};
       int resultado = sumarElementos(arreglo);
       System.out.println("La suma de los elementos es: " + resultado);
   }
}
